### ISSUE TITLE
View sreen showing code of selectbox item

### ISSUE THREAD

**Issue by hungpq12** [akeneo-labs/EnhancedConnectorBundle#26]
I used this bundle,created profiles with channel Default, but on view screen, it showing ecommerce ( channel’s code ). The same with another selectbox items of profile form. I think it should be show item’s name.

**Comment by manh-nguyen** [comment]
![profile-view-code](https://cloud.githubusercontent.com/assets/6415647/10751421/bb061d92-7cb2-11e5-9935-c1f57d30a018.png)

Screenshot for description above

**Comment by damien-carcel** [comment]
Hi hungpq12, Mahn.

I am not sure I understand your problem. You have a channel with code "ecommerce", but with label "Default", that's right?
If that's the case, then there is nothing wrong with that, and has nothing to do with the connector, it is just the normal behavior of the PIM.

Regards,
Damien

**Comment by manh-nguyen** [comment]
I meant why it's option code: doNotApply instead of option text: Do not Apply . You know, if some option codes has underscore, it looks weird  "allow_export_attributes_options"  (our extended option)

Regards,
Manh

**Comment by damien-carcel** [comment]
Hi Manh.

The option text was indeed a missing translation. It is fixed in the last 1.0.3 version.
Don't forget to clear your cache and to reload assets after update:

```
php app/console cache:clear --env=prod
php app/console pim:install:assets --env=prod
```

Regards,
Damien

**Comment by manh-nguyen** [comment]
Thanks for the update @damien-carcel !

I will update the latest code and check it.

Cheers,
Manh
